projects.json

```diff
@@ -412,8 +412,8 @@
     "path": "Guitar",
     "branch": "master",
     "compatibility": {
-      "3.1": {
```

Comment:
I'd like to maintain compatibility testing for 3.1. Could you please add it back, so we have both 3.1 and 4.0 hashes?

Reply:
sure - will do it soon.